runelite-client/src/main/java/net/runelite/client/ui/FontManager.java

```diff
@@ -36,37 +39,53 @@
 	private static final Font runescapeSmallFont;
 	private static final Font runescapeBoldFont;
 
+	// By using a 2way map we're not dependant on the toString() method so we have control over the font naming in the UI.
+	private static final HashMap<String, Font> fontLookupMap;
```

Comment:
I dont think 2 maps are necessary at all for this

Reply:
I found this: https://stackoverflow.com/questions/13236927/sorting-a-guava-bimap